src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null || repeatPassword == null) {
```

Comment:
let's move all the conditions into 1 `if` and think how we can reduce the number of conditions